Source/WebInspectorUI/UserInterface/Views/ConsoleGroup.js

```diff
@@ -34,10 +34,12 @@ WI.ConsoleGroup = class ConsoleGroup extends WI.Object
         super();
 
         this._parentGroup = parentGroup;
+        this._element = false;
```

Comment:
i think this should start out as `null` since it's going to be an object later not a boolean